ambry-api/src/main/java/com.github.ambry/server/HealthReport.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.server;
+
+/**
+ * Health report for reporting various stats within a node.
+ */
+public interface HealthReport {
```

Comment:
I agree changing the name to AmbryHealthReport. However, if we implement HealthReportProvider (helix library) then we are forcing the dependency on Helix even for a static cluster. Is that ok?